src/ingestion-server/server/images/nginx/Dockerfile

```diff
@@ -1,5 +1,5 @@
 ARG PLATFORM_ARG
-FROM --platform=$PLATFORM_ARG public.ecr.aws/docker/library/nginx:1.22
+FROM --platform=$PLATFORM_ARG public.ecr.aws/docker/library/nginx:1.25
```

Comment:
let's use stable version 1.24

http://nginx.org/en/download.html

Reply:
fixed and tested all sink types.( S3, MSK and KDS)